contracts/modules/SSVViews.sol

```diff
@@ -201,6 +201,10 @@ contract SSVViews is ISSVViews {
         return SSVStorageProtocol.load().validatorsPerOperatorLimit;
     }
 
+    function getNetworkValidatorsCount() external view override returns (uint32) {
+        return SSVStorageProtocol.load().daoValidatorCount;
```

Comment:
Just so we are aligned, this will not include liquidated validators

Reply:
correct, this does not include liquidated validators